### ISSUE TITLE
Absorb list into the openwhisk.github.io and delete this repo.

### ISSUE THREAD

**Issue by mrutkows** [openwhisk/awesome-openwhisk#11]
We will want to absorb this list into the openwhisk.github.io and delete this repo.

**Comment by mrutkows** [comment]
Of course, please choose to preserve this on your private account (repo) @jthomas

**Comment by jthomas** [comment]
That works, thanks @mrutkows.
I don't have the correct permissions on the org to transfer this back to myself. 
Can someone add me to the correct team or make me repo owner? @mbehrendt @csantanapr

**Comment by perryao** [comment]
@jthomas do you guys still plan to do this?

**Comment by csantanapr** [comment]
I think is a good idea thanks for the reminder

**Comment by csantanapr** [comment]
Well move it to a page on the website openwhisk.org
